app/src/main/java/com/jerboa/db/AppDB.kt

```diff
@@ -173,7 +182,14 @@ class AccountRepository(private val accountDao: AccountDao) {
 
 // Declares the DAO as a private property in the constructor. Pass in the DAO
 // instead of the whole database, because you only need access to the DAO
-class AppSettingsRepository(private val appSettingsDao: AppSettingsDao) {
+class AppSettingsRepository(
+    private val appSettingsDao: AppSettingsDao,
+    private val httpClient: OkHttpClient = OkHttpClient(),
+    private val ioDispatcher: CoroutineDispatcher = Dispatchers.IO,
+) {
+
+    private val _changelog = MutableStateFlow("")
```

Comment:
No reason to use a stateflow here, just use mutablestate. Its also the convention I've gone with in the `autogen_types` branch, in all the viewModels.

Reply:
Do you use `State<T>` inside a `Repository`? That is new to me. I will update the branch later.